libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -58,6 +60,52 @@ const TeamsMeetingEndT = z
     })
     .nonstrict();
 
+const TeamsMeetingParticipantsAddedT = z
+    .object({
+        Id: z.string(),
+        JoinUrl: z.string(),
+        MeetingType: z.string(),
+        Title: z.string(),
+        ParticipantsAdded: z.array(
+            z.object({
```

Comment:
This is pretty much identical to the participant structure below. Perhaps DRY this up by extracting to `ParticipantT`. Also may be worth throwing in `nonstrict()` on this object as well.

Reply:
Ah right, good catch. Thanks for the suggestion.